Client/Frontend/Browser/FirefoxTabContentBlocker.swift

```diff
@@ -65,6 +63,8 @@ class FirefoxTabContentBlocker: TabContentBlocker, TabContentScript {
         setupForTab()
     }
 
+    func prepareForDeinit() {}
```

Comment:
These empty methods feel so weird to me. I assume they are required but not used, can we improve this?